src/ds/ds-color-common.cpp

```diff
@@ -0,0 +1,134 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#include "ds/ds-color-common.h"
+#include "ds-private.h"
+#include "ds5/ds5-color.h"
```

Comment:
Same, the specific should include the common right? and the common is pure common